build.sbt

```diff
@@ -1,7 +1,23 @@
 lazy val scala212 = "2.12.20"
-ThisBuild / crossScalaVersions := Seq(scala212)
+lazy val scala3 = "3.7.2"
+
+ThisBuild / crossScalaVersions := Seq(scala212, scala3)
 ThisBuild / scalaVersion := scala212
 
+ThisBuild / (pluginCrossBuild / sbtVersion) := {
+  scalaBinaryVersion.value match {
+    case "2.12" => "1.5.8"
```

Comment:
```suggestion
    case "2.12" => "1.9.9"
```

If this plugin wants to run tests on JDK 21, then the minimum version should be 1.9.9.